src/snowflake/snowpark/_internal/compiler/large_query_breakdown.py

```diff
@@ -132,7 +155,7 @@ def apply(self) -> List[LogicalPlan]:
                 self.session.session_id,
                 SkipLargeQueryBreakdownCategory.ACTIVE_TRANSACTION.value,
             )
-            return self.logical_plans
+            return LargeQueryBreakdownResult(self.logical_plans, [])
```

Comment:
i see for the SkipLargeQueryBreakdown, it is sent as a separate telemetry, i think we can merge those, you can return {SkipLargeQueryBreakdownCategory.ACTIVE_TRANSACTION.value, 0} as s summary here, then we just need on telemetry at one place